buildSrc/src/main/java/com/synopsys/integration/detect/docs/content/Terms.java

```diff
@@ -10,10 +10,10 @@ public Terms() {
         termMap.put("solution_name", "Synopsys Detect");
         termMap.put("script_repo_url_bash", "https://detect.synopsys.com/detect9.sh");
         termMap.put("script_repo_url_powershell", "https://detect.synopsys.com/detect9.ps1");
-        termMap.put("binary_repo_url_project", "https://sig-repo.synopsys.com/bds-integrations-release/com/synopsys/integration/synopsys-detect");
-        termMap.put("binary_repo_ui_url_project", "https://sig-repo.synopsys.com/ui/repos/tree/General/bds-integrations-release/com/synopsys/integration/synopsys-detect");
-		termMap.put("binary_repo_jenkins_url_project", "https://sig-repo.synopsys.com/artifactory/bds-integrations-release/com/blackducksoftware/integration/blackduck-detect/");
-		termMap.put("binary_repo_url_sigma", "https://sig-repo.synopsys.com/artifactory/sigma-release-trial/2022.6.0/");
+        termMap.put("binary_repo_url_project", "https://repo.blackduck.com/bds-integrations-release/com/blackduck/integration/synopsys-detect");
+        termMap.put("binary_repo_ui_url_project", "https://repo.synopsys.com/bds-integrations-release/com/synopsys/integration/synopsys-detect");
```

Comment:
@gopannabd This should be repo.blackduck and not repo.synopsys right?

Reply:
Corrected, thanks!